app/models/ext_management_system.rb

```diff
@@ -45,6 +45,8 @@ def self.supported_types_and_descriptions_hash
   has_many :storages,       -> { distinct },          :through => :hosts
   has_many :ems_events,     -> { order "timestamp" }, :class_name => "EmsEvent",    :foreign_key => "ems_id",
                                                       :inverse_of => :ext_management_system
+  has_many :carried_events, -> { order "timestamp" }, :class_name => "EmsEvent",    :foreign_key => "source_ems_id",
```

Comment:
as naming is hard, let's reflect a bit on naming here 😄 it's hard to reverse later.

To me `carried_events` doesnt immediately ring a bell. Could you explain in prose what those events are? Maybe a better name emerges then...